src/lib/modules/executor/executor.go

```diff
@@ -0,0 +1,105 @@
+package executor
```

Comment:
```suggestion
package evaluate
```

let's try with `evaluate` and see how the code looks.